index.php

```diff
@@ -68,105 +53,88 @@ function myErrorHandler($errno, $errstr, $errfile, $errline)
             break;
         case E_STRICT:
             $errors = "Strict Notice";
-            break; // catched only outside this file
+            break;
         case E_RECOVERABLE_ERROR:
             $errors = "Recoverable Error";
             break;
         default:
-            if (defined('E_DEPRECATED') and $errno == E_DEPRECATED) {
+            if ($errno == E_DEPRECATED) {
                 $errors = "Deprecated";
-            } elseif (defined('E_USER_DEPRECATED') and $errno == E_USER_DEPRECATED) {
+            } elseif ($errno == E_USER_DEPRECATED) {
                 $errors = "User Deprecated";
             } else {
                 $errors = "Unknown error ($errno)";
             }
-            break;
     }
 
-  // Store error, to print it later
-  #$err = '<b>'. $errors .'</b>: '. $errstr .' in <b>'. $errfile .'</b> on line <b>'. $errline .'</b><br /><br />';
     $err = sprintf("PHP %s: %s in %s on line %d", $errors, $errstr, $errfile, $errline);
-
-  // Write error to log file
     if (ini_get('log_errors')) {
         error_log($err);
     }
 
-  // Write to $PHPError for onscreen output later
     $PHPErrors .= $err .'<br />';
     $PHPErrorsFound = 1;
 
-  // Write to DB-Log
-  // Attention: Be aware of loops!
+    // Write to DB-Log
     if (isset($db) and $db->success) {
-        $db->qry(
-            'INSERT INTO %prefix%log
-        SET date = NOW(), userid = %int%, type = 3, description = %string%, sort_tag = "PHP-Fehler"',
-            (int)$auth['userid'],
+        $db->qry('
+            INSERT INTO %prefix%log
+            SET date = NOW(),
+                userid = %int%,
+                type = 3,
+                description = %string%,
+                sort_tag = "PHP-Fehler"',
```

Comment:
Should we change this from "PHP-Fehler" to "PHP-Error" or maybe a general "PHP"?
Did a quick search and this is the only place in the source code where it is used.

Field is only used in the Log viewer for sorting purposes and the selection list is auto-filled based on unique entries, so no separate edit needed there..

Reply:
True. This needs to be investigated further what it is used to be. For this i created #115